packages/lesswrong/server/fmCrosspost/crosspost.ts

```diff
@@ -155,11 +147,16 @@ export async function handleCrosspostUpdate(
     logger('post is not a crosspost, returning')
     return data;
   }
+  if (!fmCrosspost?.hostedHere) {
+    logger('post is not a hosted here, returning')
+    return data;
+  }
```

Comment:
I would have thought that, once we allow editing the post on both sites, edits on the foreign site would get propagated to the host site? I think it would be surprising if it worked in one direction and not the other